app/ui/src/app/integration/import-export/import/integration-import.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { PlatformStore } from '@syndesis/ui/platform';
+
+
+import {
+  IntegrationImportUpload
+} from './integration-import.actions';
+
+@Component({
+  selector: 'syndesis-import-integration-component',
+  templateUrl: './integration-import.component.html',
+  styleUrls: ['./integration-import.component.scss']
+})
+export class IntegrationImportComponent implements OnInit {
+  constructor(private store: Store<PlatformStore>) { }
+
+  ngOnInit() {
+    /**
+     * Payload for IntegrationImportUpload action
+     */
```

Comment:
Why do we need a comment for this? Isn't the variable names or the method naming self-descriptive enough?